src/frontends/onnx/frontend/src/ops_bridge.cpp

```diff
@@ -115,6 +115,7 @@
 #include "op/min.hpp"
 #include "op/mod.hpp"
 #include "op/mul.hpp"
+#include "op/mish.hpp"
```

Comment:
It should keep alphabetical order (required by style check):
```suggestion
#include "op/mish.hpp"
#include "op/mod.hpp"
#include "op/mul.hpp"
```